ansible/vars/epoch/main.yml

```diff
@@ -105,7 +105,6 @@ epoch_config:
 
   mining:
     autostart: false
-    beneficiary: "ak_tjnw1KcmnwfqXvhtGa9GRjanbHM3t6PmEWEWtNMM3ouvNKRu5"
```

Comment:
`autostart: false` above is redundant

Reply:
I know but I don't want to rely on defaults ;)